backend/src/main/java/club/devcord/devmarkt/services/UserService.java

```diff
@@ -55,6 +58,13 @@ public boolean delete(UserId userId) {
     return repo.deleteByUserId(userId) >= 1;
   }
 
+  public User createDefaultUserUnsafe(UserId userId) {
+    var user = new User(-1, userId, Set.of(new Role(-1, Roles.USER.toString())));
+    repo.save(user);
+    repo.addRoles(userId, Set.of(Roles.USER.toString()));
```

Comment:
Why do we need to add the role to the user if the role has already been set in the constructor?

Reply:
The "save" method does not insert the roles, which is due to the join table used here. So we need to add them manually.